src/bun.js/bindings/KeyObject.cpp

```diff
@@ -2943,6 +2947,150 @@ JSC_DEFINE_HOST_FUNCTION(KeyObject__SymmetricKeySize, (JSC::JSGlobalObject * glo
     return JSC::JSValue::encode(JSC::jsUndefined());
 }
 
+static EncodedJSValue doAsymmetricCipher(JSGlobalObject* globalObject, CallFrame* callFrame, bool encrypt)
+{
+    auto count = callFrame->argumentCount();
+    auto& vm = globalObject->vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    if (count != 2) {
+        return Bun::throwError(globalObject, scope, Bun::ErrorCode::ERR_MISSING_ARGS,
+            "expected object as first argument"_s);
+    }
+
+    auto* jsKey = jsDynamicCast<JSObject*>(callFrame->uncheckedArgument(0));
+    if (!jsKey) {
+        return Bun::throwError(globalObject, scope, Bun::ErrorCode::ERR_INVALID_ARG_TYPE,
+            "expected object as first argument"_s);
+    }
+
+    auto jsCryptoKeyValue = jsKey->getIfPropertyExists(
+        globalObject, PropertyName(Identifier::fromString(vm, "key"_s)));
```

Comment:
I think we can use `BuiltinNames` for this one:
```suggestion
        globalObject, builtinNames(vm).keyPublicName());
```